### PR TITLE
Document "Getting client ID and client secret"

### PR DESCRIPTION
**Pull request by wontheone1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,12 @@ Yarn:
 yarn add sharetribe-flex-integration-sdk
 ```
 
+## Getting client ID and client secret
+
+Client ID and secret are required to create new SDK instance. 
+See [the doc](https://www.sharetribe.com/docs/integrations/getting-started-with-integration-api/#create-integration-api-application-in-flex-console)
```
Comment (bladealslayer) [inline README.md:37]: I'd perhaps just move this under the Usage section and below the code example. So no separate heading.

File: README.md [hunk 2]
```diff
@@ -31,6 +31,12 @@ Yarn:
 yarn add sharetribe-flex-integration-sdk
 ```
 
+## Getting client ID and client secret
+
+Client ID and secret are required to create new SDK instance.
```
Comment (bladealslayer) [inline README.md:36]:
```suggestion
Client ID and client secret are required to create a new SDK instance. 
```

File: README.md (hunk 1 above)
Comment (wontheone1) [reply README.md]: Changed as suggested

File: README.md (hunk 2 above)
Comment (wontheone1) [reply README.md]: Changed as suggested

**Comment by wontheone1** [comment]
@bladealslayer @lyyder Since this is rather a simple PR, I will merge without waiting for another approval. Feel free to suggest more after merging.

**Pull request merged**